.drone.yml

```diff
@@ -3,98 +3,81 @@ kind: pipeline
 name: test
 
 steps:
+  - name: fetch
+    image: docker:git
+    commands:
+      - git fetch --tags
+
   - name: lint
     image: golang
     pull: always
+    depends_on:
+      - fetch
     commands:
       - find . -name "*.go" | xargs gofmt -s -d
-      - go vet ./...
+      #- go vet ./...
     when:
       event:
         - push
         - tag
-      
-  - name: build-linux-push
-    group: build
+
+  - name: go-mod-vendor
     image: golang
     pull: always
-    depends_on: 
+    depends_on:
       - lint
-    environment:
-      GOOS: linux
-      GOARCH: amd64
-      CGO_ENABLED: 0
     commands:
       - go mod vendor
-      - go build
     when:
       event:
         - push
 
-  - name: build-darwin-push
+  - name: build-linux-push
     group: build
     image: golang
     pull: always
-    depends_on: 
-      - lint
+    depends_on:
+      - go-mod-vendor
     environment:
-      GOOS: darwin
+      GOOS: linux
       GOARCH: amd64
-      CGO_ENABLED: 0    
+      CGO_ENABLED: 0
     commands:
-      - go mod vendor
-      - go build 
+      - go build
     when:
       event:
         - push
 
-  - name: build-linux
-    group: build-to-release
+  - name: build-darwin-push
+    group: build
     image: golang
     pull: always
-    depends_on: 
-      - lint
+    depends_on:
+      - go-mod-vendor
     environment:
-      GOOS: linux
+      GOOS: darwin
       GOARCH: amd64
       CGO_ENABLED: 0
     commands:
-      - go mod vendor
-      - go build -a -ldflags '-extldflags "-static"' -o bin/releases/furyctl-linux-amd64  .
+      - go build
     when:
       event:
-        - tag
+        - push
 
-  - name: build-darwin
-    group: build-to-release
-    image: golang
+  - name: build-release
+    image: goreleaser/goreleaser:v0.124
     pull: always
-    depends_on: 
+    depends_on:
       - lint
+      - fetch
     environment:
-      GOOS: darwin
-      GOARCH: amd64
-      CGO_ENABLED: 0    
+      GITHUB_TOKEN:
+        from_secret: GITHUB_TOKEN
+    secrets:
+      - GITHUB_TOKEN
     commands:
-      - go mod vendor
-      - go build -a -ldflags '-extldflags "-static"' -o bin/releases/furyctl-darwin-amd64 .
+      - goreleaser check
+      - goreleaser --debug
```

Comment:
Do we really need the `--debug`?

Reply:
not really. But can help in case something will go wrong for  finding root cause.

Reply:
👍